### ISSUE TITLE
Question about npt simulation

### ISSUE THREAD

**Issue by Chenghao-Wu** [jax-md/jax-md#294]
Hi JAX-MD developers,

I noticed that in the notebook: npt_simulation, it states that "For now NPT simulations must be performed with fractional coordinates, where the atom positions are stored in the unit cube. This restriction can likely be relaxed in the future, if it were desirable." 

Is it still true for the latest version?

**Comment by sschoenholz** [comment]
Yeah, it's still true. The basic reason is that it's more challenging to resize the unit cell if atom positions are not stored fractionally. I'm sure it's possible to do this by e.g. projecting into the unit cube and then back into the new space, but it's a bit tricky to get right.
